messages/pl.json

```diff
@@ -58,5 +58,25 @@
       "placeholder": "Wybierz"
     },
     "submit": "Dodaj"
+  },
+  "DefaultCategories": {
+    "mortgage": "Kredyt",
```

Comment:
kredyt a hipoteka to co innego lekko, generyczniej byłoby może zamienić mortgage na loan